### ISSUE TITLE
Prototype Pollution vulenrability

### ISSUE THREAD

**Issue by andrew-itscript** [facebook/facebook-nodejs-business-sdk#192]
### Which SDK version are you using?
^12.0.1

### What's the issue?
Snyk reporting Prototype Pollution vulenrability

### Steps/Sample code to reproduce the issue
https://app.snyk.io/test/npm/facebook-nodejs-business-sdk/12.0.1

#### Observed Results:
![snyk_issue](https://user-images.githubusercontent.com/29883742/113968147-bbcb2d00-985c-11eb-8636-301832ab0e9c.png)

#### Expected Results:
No vulnerabilities should be found by Snyk

**Comment by andrew-itscript** [comment]
it still requires the community's attention. Prototype Pollution vulenrability is not fixed. Please see
 ```
 Prototype Pollution [High Severity][https://snyk.io/vuln/SNYK-JS-MOUT-1014544] in mout@0.11.1
 introduced by facebook-nodejs-business-sdk@12.0.1 > iso-3166-1-alpha-2@1.0.0 > mout@0.11.1
```

**Comment by jfaylon** [comment]
https://github.com/advisories/GHSA-pc58-wgmc-hfjr for more info. Still an issue.

**Comment by remarkablemark** [comment]
Still an issue:

```
# npm audit report

mout  <1.2.3
Severity: high
Prototype Pollution in mout - https://github.com/advisories/GHSA-pc58-wgmc-hfjr
fix available via `npm audit fix --force`
Will install facebook-nodejs-business-sdk@6.0.0, which is a breaking change
node_modules/mout
  iso-3166-1-alpha-2  *
  Depends on vulnerable versions of mout
  node_modules/iso-3166-1-alpha-2
    facebook-nodejs-business-sdk  >=6.0.1
    Depends on vulnerable versions of iso-3166-1-alpha-2
    node_modules/facebook-nodejs-business-sdk
```

**Comment by andrew-ignatiev** [comment]
Is it possible to prioritize fixing of this vulnerability as it impacts negatively on cybersecurity report and business?

**Comment by StephaneBischoff** [comment]
Any updates? 
![image](https://user-images.githubusercontent.com/9750058/166823069-66239711-4e8d-4ab1-9eb5-f03f8607793c.png)

**Comment by andrew-ignatiev** [comment]
@facebook-github-bot why the issue is closed when PR merge was failed ? Could you drop support of Node.js version 8?

**Comment by Dezzymei** [comment]
npm's automated fix seems to cause a rollback to version 6.0.0

It would be great to remove this, please!
